README.md

```diff
@@ -53,6 +53,9 @@ OPTIONS:
     -t, --threshold <THRESHOLD>
             Sets the maximum amount of space to be used for Docker images (default: 10 GB)
 
+    -m, --min-age <Duration>
```

Comment:
For consistency with the other options: `DURATION` instead of `Duration`

Reply:
Fixed